pkg/plugins/resources/yaml/target.go

```diff
@@ -82,12 +87,13 @@ func (y *Yaml) target(source string, dryRun bool) (bool, []string, string, error
 	if !dryRun {
 
 		newFile, err := os.Create(y.spec.File)
-		defer newFile.Close()
```

Comment:
What is the reason to remove this line?

Reply:
Moved to L95

Reply:
yep: the line was moved. But the new position is a file handler leak: if the error is not null BUT the file handler is still opened (even in memory: case of a read only filesystem for instance), then the app will leak because the handler won't be closed even after the execution, and only a reboot of the instance would clean it up

Reply:
I added an instruction to ignore the linter check

Reply:
Interesting, the static lint check allows to make sure that my assertion was wrong. Sounds like golang id able to catch the case I described during compilation, so your change made sense (to move the line) as it implies a better code readability. Don’t hesitate to explain sich change with a static link next time